test/tools/nl_files/subset_data_region1

```diff
@@ -1 +1 @@
-region --lat1 -40 --lat2 15 --lon1 275 --lon2 330 --create-domain --create-surface --create-landuse --verbose --overwrite --reg test1 --inputdata-dir CSMDATA
```

Comment:
Since mesh_masker.py was added as a new script.  We should at least add a tools test for it.

Reply:
I added both tools test and python unit and sys tests.